modules/core/src/main/java/org/apache/ignite/configuration/DataStorageConfiguration.java

```diff
@@ -337,6 +337,14 @@ public class DataStorageConfiguration implements Serializable {
     /** Default memory allocator for all data regions. */
     @Nullable private MemoryAllocator memoryAllocator = null;
 
+    /** Flag to enable or disable PITR (Point In Time Recovery) feature. */
+    @IgniteExperimental
+    private boolean pointInTimeRecoveryEnabled;
```

Comment:
All `pointInTimeRecoveryXXX` could be replaced with `pitrXXX` because it's a well known abbreviation.